java/ci/README.md

```
@@ -1,11 +1,100 @@
+```bash
+./java/ci/build_cudf_java_jar.sh \
+  --libcudf-dir /tmp/libcudf-cuda12 \
+  --output-dir /tmp/jars \
+  --cuda-version 12.9
+```
+
+This compiles the JNI layer against the static libcudf from Step 1 and emits a
+single classifier JAR (e.g. `cudf-26.08.0-SNAPSHOT-cuda12.jar`) plus its POM
+into a classifier-named subdirectory under `--output-dir`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)